pkg/render/node.go

```diff
@@ -644,15 +644,20 @@ func (c *nodeComponent) cniContainer() v1.Container {
 		{MountPath: "/host/etc/cni/net.d", Name: "cni-net-dir"},
 	}
 
+	command := "/install-cni.sh"
+
 	image := components.GetReference(components.ComponentCalicoCNI, c.cr.Spec.Registry, c.cr.Spec.ImagePath)
 	if c.cr.Spec.Variant == operator.TigeraSecureEnterprise {
 		image = components.GetReference(components.ComponentTigeraCNI, c.cr.Spec.Registry, c.cr.Spec.ImagePath)
+		// Enterprise release-v3.2 has cni changes that converted shell script to golang binary. The same change
+		// is not in the corresponding os version.
+		command = "/opt/cni/bin/install"
```

Comment:
I appreciate the fix you've done here. but why isn't this in OS?

Could we just PR a change to cni-plugin which runs `ln -s /install-cni.sh /opt/cni/bin/install` during the image build until it is added there too?

Reply:
It is in OS but not in a released version of OS.

Reply:
We could change the cni-plugin but it is simpler to just change what we're deploying instead of adding cruft to the cni-plugin.

Reply:
I picked the changes that are in for the v3.2 enterprise release. I think we should update the images to match what it uses when it is released. So I thought it would be best to just go with what it has.

Reply:
OH I didn't notice the base branch on this PR was our feature branch. LGTM